pages/communication_production/batch_list_page.py

```diff
@@ -204,6 +207,17 @@ def select_first_batch_row(self, table_selector: str, timeout_ms: int = 0) -> No
         ), f"No batch links found in table '{table_selector}'"
         first_link.click()
 
+    def navigate_to_active_batch_list_page(self) -> None:
+        """
+        Navigates to the active batch list page from anywhere in bcss (providing the main menu link is displayed).
+
+        Args:
+            page (Page): The Playwright page object.
+        """
+        BasePage(self.page).click_main_menu_link()
+        BasePage(self.page).go_to_communications_production_page()
```

Comment:
Since this class inherits BasePage I think `self.click_main_menu_link()` can be used instead of `BasePage(self.page).click_main_menu_link()`
```suggestion
        self.click_main_menu_link()
        self.go_to_communications_production_page()
```

Reply:
Also no need to add page under arguments as this is not massed into the method but instead when you initialise the class

Reply:
Updated calls and removed page from args